src/code.cloudfoundry.org/policy-server/cmd/policy-server/main.go

```diff
@@ -437,6 +447,16 @@ func main() {
 		{"debug-server", debugServer},
 	}
 
+	if conf.ASGSyncInterval > 0 {
```

Comment:
Should we either inspect ASGSyncEnabled, or have ASGSyncInterval set to 0 when ASGSyncEnabled is false? Right now if ASGSyncEnabled is false, by default, ASGSyncInterval is still 60, right? :)

Reply:
It defaults to 0 if not set in the config (when the disable property is set), but we should move this to an explicit enable/disable hooked up to the bosh property, and require a sync interval when it's enabled.